nuprojs/Microsoft.ServiceFabric.Services.Remoting/Microsoft.ServiceFabric.Services.Remoting.nuproj

```diff
@@ -98,5 +98,8 @@
       <Dependency Include="Microsoft.ServiceFabric.FabricTransport.Internal">
         <Version>[$(NugetPkg_Version_Microsoft_ServiceFabric_FabricTransport_Internal)]</Version>
       </Dependency>
+	  <Dependency Include="System.Diagnostics.DiagnosticSource">
+		<Version>[5.0.0.0]</Version>
```

Comment:
Need framework dependent nuget inclusion

Reply:
Just trying to understand, does framework here mean between netcore and netframework? or between net5.0 and netcore3.1 such